deploy/charts/rook-ceph-cluster/values.yaml

```
@@ -448,6 +495,13 @@ cephObjectStores:
       preservePoolsOnDelete: true
       gateway:
         port: 80
+        resources:
+          limits:
+            cpu: "500m"
+            memory: "512Mi"
+          requests:
+            cpu: "500m"
```

How about higher default of 1cpu? Similar changes for the limits as well
```suggestion
            cpu: "1000m"
```